custom_components/ha-google-home/sensor.py

```diff
@@ -60,8 +78,11 @@ def __init__(self, coordinator, entry, device):
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
+        _LOGGER.debug("ALARM ATTRIBUTE data", self.coordinator.data)
```

Comment:
You're not seeing any data here because logger supports only 1 argument unlike `print`. It should be: `_LOGGER.debug(f"ALARM ATTRIBUTE data {self.coordinator.data}")

Reply:
It's not very precise, actually logger supports format string and extra arguments: https://docs.python.org/3/library/logging.html#logging.debug
But anyway `coordinator.data` is there, you only need to process it.